src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -2669,372 +2587,363 @@
-  "botania.page.corporea2": "When a $(1)Corporea Spark$(0) is placed or polled, each spark will poll any nearby sparks, adding them to the network designated by the single $(1)Master Corporea Spark$(0) and also polling other nearby sparks, in a chain. While each individual spark can only reach a few blocks, combined, the entire network can go for a virtually infinite distance.",
-  "botania.page.corporea3": "$(1)Corporea Sparks$(0) have a view of the inventory directly below them, but can only access items from the top side. Each $(1)Corporea Spark$(0) has the ability to view every other spark in the network and its contents, and the ability to request them as necessary, depending on the type of block below it.",
-  "botania.page.corporea4": "It's to note that the $(1)Master Corporea Spark$(0), while being part of the network, can not have the items in its inventory extracted from.$(p)Futhermore, no requests for items in the network are ever done passively. Corporea does not have the ability to add new items to the network, those should be inserted physically through $(1)Hoppers$(0) or other methods.",
-  "botania.page.corporea5": "Similarly to normal $(1)Sparks$(0), $(1)Corporea Sparks$(0) have a method of augmentation, that being the ability to color them. While in the world, there's an icon floating around the spark that shows which color it has, this can be changed by right clicking on the spark with a piece of $(1)Floral Powder$(0). These colors define networks. A spark may only be in the same network as another spark if the two have the same color.",
-  "botania.page.corporea6": "The ability to color sparks allows for multiple different $(4)Corporea Networks$(0) to be together in a close area, without interfering with each other.$(p)Viewing the network and removing the spark works exactly the same as a normal spark. Lastly, having two $(1)Master Corporea Sparks$(0) of the same color nearby, while possible, is a bad idea and will cause conflicts and odd behaviour.",
+  "botania.page.corporea0": "The $(4)Corporea$(0) system uses the energy of the $(4)End$(0), and the void between worlds, to transport items from point A to point B instantly and without blocks to connect them.$(p)The system, at its core, uses variant $(1)Sparks$(0) (called $(1)Corporea Sparks$(0)) that transport items instead of $(4)Mana$(0).",
+  "botania.page.corporea1": "A few things to know: $(1)Corporea Sparks$(0) are placed on top of $(4)Inventories$(0) (like a $(1)Chest$(0) or $(1)Dispenser$(0)), as opposed to $(4)Mana Containers$(0). There are $(o)two$() types of $(1)Corporea Sparks$(0): the plain old $(1)Corporea Spark$(0) and the $(1)Master Corporea Spark$(0). These sparks work in networks: each $(4)Corporea Network$(0) needs one $(1)Master Corporea Spark$(0), but $(o)any$() spark can communicate with any other in the network.",
+  "botania.page.corporea2": "When a $(1)Corporea Spark$(0) is placed or right-clicked, each spark in the networ
```

Comment:
Actually this entire part can be deleted as vanilla added this a good while ago and the entry no longer exists either.